content/docs/user-guide/components/reference/atom/hdr-color-grading.md

```diff
@@ -0,0 +1,68 @@
+---
+title: HDR Color Grading Component
+linktitle: HDR Color Grading
+description: 'Open 3D Engine (O3DE) HDR Color Grading component reference.'
+toc: true
+---
+
+The **HDR Color Grading** component is a post-process effect that enables users to color grades scenes matching a desired look and feel. 
+This component uses a shader behind the scenes to perform the color grading logic on each frame's pixel. To reduce compute time, a Look-up (LUT) texture can be generated from the current color grading settings by clicking `Generate LUT` on the component. 
+![HDR Color Grading Generate LUT](/images/user-guide/components/reference/atom/post-processing-modifiers/hdr-color-grading-generate-lut-ui.png)
+
+The generated LUT texture can then be activated by clicking on the `Activate LUT` button on the component. 
+![HDR Color Grading Activate LUT](/images/user-guide/components/reference/atom/post-processing-modifiers/hdr-color-grading-activate-lut-ui.png)
+
+Upon activation, a **Look Modification** component is added to the current entity and subsequently the **HDR Color Grading** component is disabled. 
+
+## Provider ##
+
+[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
+
+## Dependencies
+
+[PostFX Layer component](/docs/user-guide/components/reference/atom/postfx-layer/)
+
+## Base properties
+
+![HDR Color Grading base properties](/images/user-guide/components/reference/atom/post-processing-modifiers/hdr-color-grading-component-ui.png)
+
+| Property | Description | Recommended Values | Default |
+|-|-|-|-|
+| Color Adjustment |
+| **Weight** | Strength of the color adjustment properties | `0.0` to `1.0` | `1.0` |
```

Comment:
Need a period at the end of each sentence for all properties.

Reply:
Fixed.